src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -714,6 +715,55 @@ export class SavedObjectsRepository {
     return body;
   }
 
+  /**
+   * Deletes all objects from the provided workspace. It used when deleting a workspace.
```

Comment:
bit confused by this comment - `It used when deleting a workspace.` ?

Reply:
For now we don't have an action on UI to only delete saved objects belongs to a workspace and not delete workspace itself. This may used in other case in future, let me remove it to avoid confusing.